guides/source/developers/promotions/promotion-actions.html.md

```diff
@@ -82,7 +82,7 @@ You must then register the custom action in an initializer in your
 `config/initializers/` directory:
 
 ```ruby
-Rails.application.config.spree.promotions.actions << MyPromotionAction
+Rails.application.config.spree.promotions.actions << Spree::Promotion::Actions::MyPromotionAction
```

Comment:
I don't believe this is a pattern we wish to encourage. If this has resolution issues as it is I think that:

```Ruby
Rails.application.config.spree.promotions.actions << ::MyPromotionAction
```

or 


```Ruby
Rails.application.config.spree.promotions.actions << MyNamespace::MyPromotionAction
```

would be preferrable.

Reply:
What I suggested is what worked for me, but if either one of these two alternatives resolve as well and the pattern is preferred, sure, why not.